### PR TITLE
Header

### PR DESCRIPTION
**Pull request by brimorales**
Header feature: LoopCommerce logo, Searchbar, Login, Sign up and My Cart buttons.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -9,4 +9,7 @@ def update
   def user_params
     params.require(:user).permit(:avatar)
   end
+  def search
```
Comment (diebas) [inline app/controllers/users_controller.rb]: add a space between methods

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -9,4 +9,7 @@ def update
...
+    @user = User.search(params[:search])
```
Comment (diebas) [inline app/controllers/users_controller.rb]: what is this search for?

Comment (diebas) [reply app/controllers/users_controller.rb]: remember the search bar on the header is ment for finding products, not users

Comment (brimorales) [reply app/controllers/users_controller.rb]: That's something I forgot to delete. Should I delete it and push it again?

Comment (diebas) [reply app/controllers/users_controller.rb]: yes! it's better to avoid merging unnecessary code to develop

**Review by diebas — APPROVED** [review]
good job! It would be great if in the next prs you add screenshots of the feature in the description.

**Pull request merged**